lib/model/KnowledgePanelElement.dart

```diff
@@ -180,6 +180,57 @@ class KnowledgePanelTableColumn extends JsonObject {
   Map<String, dynamic> toJson() => _$KnowledgePanelTableColumnToJson(this);
 }
 
+/// Element representing a world map.
+@JsonSerializable()
+class KnowledgePanelWorldMapElement extends JsonObject {
+  final List<KnowledgePanelGeoPointer> pointers;
+
+  const KnowledgePanelWorldMapElement({
+    required this.pointers,
+  });
+
+  factory KnowledgePanelWorldMapElement.fromJson(Map<String, dynamic> json) =>
+      _$KnowledgePanelWorldMapElementFromJson(json);
+
+  @override
+  Map<String, dynamic> toJson() => _$KnowledgePanelWorldMapElementToJson(this);
+}
+
+/// Element representing a geo location of a map pointer.
+@JsonSerializable()
+class KnowledgePanelGeoPointer extends JsonObject {
+  final KnowledgePanelLatLng? geo;
```

Comment:
Can it be `null`? What would be the point, then?

Reply:
yeah Stephane said it could have other information (non-geo). I actually don't know exactly what yet